chore: remove unnecessary suffix from @vaadin/flow-frontend imports (#131)
vaadin/docs: articles/fusion/routing/nested-views.asciidoc

== Import Theme Global Styles

In Vaadin, themes have some global styles that need to be applied to the page. 

> With server-side main layout in Java, these are automatically applied for convenience. 

Reviewer: ```suggestion
With a server-side main layout in Java, these are automatically applied for convenience. 
```

> With TypeScript main layout, though, they need to be imported manually to make sure consistent styling between server-side and client-side views.

Reviewer: ```suggestion
With a client-side main layout in TypeScript, they need to be imported manually to get consistent styling between server-side and client-side views.
```

== Establish an Application Layout

The most prominent feature of the main layout is to define the layout for the application. 

> You can use the `<vaadin-app-layout>` component:

Reviewer: ```suggestion
You can use the <<{articles}/ds/components/app-layout#,App Layout>> component:
```
